tools/integration-tests/tester/framework/parameters.go

```diff
@@ -89,6 +89,7 @@ func PeerConfig() config.GoShimmer {
 	c.Faucet.GenesisTokenAmount = 2500000000000000
 
 	c.Mana.Enabled = true
+	c.Mana.SnapshotResetTime = true
```

Comment:
How integration tests worked before without this? 
Or it's only now when the mana decayed to 0?

Reply:
The snapshot is from march, so I suppose so much time has passed that some values started rounding up to 0, which resulted in NaN. Currently we're currently investigating whether the formula to update the EffectiveBaseMana2 is also correct.